asmcli/commands/create-mesh.sh

```diff
@@ -22,18 +22,46 @@ parse_cluster_args() {
   fi
 
   while [[ $# != 0 ]]; do
-    local CLUSTER; CLUSTER="${1}"
-    context_append "clustersInfo" "${CLUSTER//\// }"
+    if [ -e "$1" ]; then
```

Comment:
We should use `-f` instead of `-e`. (see https://tldp.org/LDP/abs/html/fto.html)

Reply:
yes, thanks!